README.win32.adoc

```diff
@@ -29,7 +29,8 @@ Here is a summary of the main differences between these ports:
 (*):: Cygwin-generated `.exe` files are linked with the Cygwin DLL. Since
 version 2.5.2 released in June 2016, this DLL is licensed under the LGPLv3 with
 a static linking exception; prior versions of the DLL are distributed under the
-GPL. Thus, these `.exe` files can only be distributed under compatible license.
+GPL (the cygwin version can be obtained with the `uname -r` command). Thus,
+these `.exe` files can only be distributed under a compatible license.
```

Comment:
I agree with @oandrieu that we shouldn't be too specific here, and let the reader decide if they are compliant or not. Perhaps just recommending Cygwin 2.5.2 or higher is sufficient for this regard and let users compile against earlier versions at their risk.